openmetadata-ui/src/main/resources/ui/src/components/recently-viewed/RecentlyViewed.tsx

```diff
@@ -73,10 +73,7 @@ const RecentlyViewed: FunctionComponent = () => {
               name,
               owner: getOwnerFromId(owner?.id)?.name || '--',
               serviceType: oData.serviceType,
-              tags: [
-                getTierFromTableTags(tags),
-                ...tableTags.map((tag) => tag.tagFQN),
-              ].filter((tag) => tag),
+              tags: [...tableTags].filter((tag) => tag),
```

Comment:
@ShaileshParmar11 We would need a tier in the tag list because tier is also a tag.